index.html

```diff
@@ -647,15 +647,17 @@ <h3 id="AB">2.3 Advisory Board (AB)</h3>
       <h4 id="ABParticipation">2.3.1 Advisory Board Participation</h4>
 
       <p>The Advisory Board consists of eleven elected participants
-        and a Chair.
-        The Team appoints the Chair of the <a href="#AB">Advisory Board</a>, who is generally the CEO.
+        the CEO,
+        and Chair(s), who are generally participants of the AB.
+        The elected members of the AB and the CEO by consensus (or by vote if that fails) appoint the Chair(s) of the Advisory Board,
```

Comment:
This doesn't say anything about how frequently the AB revisits the choice of chair, which is probably a good idea in both directions - you don't want to change chairs every month, ideally, but you also should be forced to ask the question "is the current chair the optimal one" on so often a cycle that it doesn't seem epochal to choose a new chair.